src/protocol.h

```diff
@@ -26,14 +27,14 @@
 class CMessageHeader
 {
 public:
-    static constexpr size_t MESSAGE_START_SIZE = 4;
+    using MessageStartChars = MessageStartMagic;
```

Comment:
nit: I would be okay with dropping this and renaming the 6 places where we use `MessageStartChars`